packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -447,6 +447,41 @@ export const components: ThemeOptions["components"] = {
             backgroundColor: theme.palette.grey[300],
           },
         }),
+
+        ...(ownerState.variant === "status" && {
+          paddingBlock: 0,
+          paddingInline: 0,
+          borderRadius: 0,
+          backgroundColor: "transparent",
+          color: theme.palette.text.primary,
+
+          "&::before": {
+            content: "''",
+            width: ".57em",
+            height: ".57em",
```

Comment:
Why are we using `em` rather than `rem`? Does it scale better when resizing the font size?

Reply:
Answer: https://github.com/okta/odyssey/pull/1655/files#r1066369436